management/nodes/reboot/setup_steps.README

```diff
@@ -0,0 +1,41 @@
+### Prerequisites
+
+Follow the setup steps from <v-jiba@microsoft.com> to get a triple-boot machine</br>
+*Note: To connect your laptop to the same network as your master computer, you may need a wired connection if you have driver issues*
+
+### Windows
+1. Setup auto-login to the admin account
+2. Turn off sleep mode
+3. Install Jenkins through the Java Web Start button on the page for your node</br>
```

Comment:
Typically on Windows we just use the task scheduler to run Jenkins once the system logs in, then we set auto-login.